src/MfaVerifyController.js

```diff
@@ -100,6 +115,15 @@ function (Okta, Checkbox, BaseLoginController, TOTPForm, YubikeyForm, SecurityQu
           this.options.appState.get('isMfaRequired')) {
         return true;
       }
+      // update auto push cookie after user accepts Okta Verify MFA
+      if (this.options.factorType == 'push') {
+        if (this.settings.get('features.autoPush') && !!this.model.get('autoPush')) {
```

Comment:
Curious - what is the type of the value that is returned by this.model.get('autoPush')? Is it not a boolean?